src/relayclient/RelayProvider.ts

```diff
@@ -40,7 +44,11 @@ export class RelayProvider implements HttpProvider {
 
     this.origProvider = origProvider
     this.config = config
-    this.origProviderSend = this.origProvider.send.bind(this.origProvider)
+    if (typeof this.origProvider.sendAsync === 'function') {
```

Comment:
hmm. sendAsync was removed in web3-provider>1.0. so buidler implements the old protocol?

Reply:
web3-provider-engine enforces calling to 'sendAsync'